test/gtest/ucs/test_event_set.cc

```diff
@@ -12,7 +12,9 @@ extern "C" {
 
 #define MAX_BUF_LEN 255
 
-static const char *UCS_EVENT_SET_TEST_STRING = "ucs_event_set test string";
+static const char* UCS_EVENT_SET_TEST_STRING  = "ucs_event_set test string";
+static const char* UCS_EVENT_SET_EXTRA_STRING = "ucs_event_set extra string";
+static const int   UCS_EVENT_SET_EXTRA_NUM    = 0xFF;
```

Comment:
Do I need an alignment? 

Which is preferable?

A

```C
static const char *UCS_EVENT_SET_EXTRA_STRING = "ucs_event_set extra string";
static const int  UCS_EVENT_SET_EXTRA_NUM     = 0xFF;
```

B 

```C
static const char *UCS_EVENT_SET_EXTRA_STRING = "ucs_event_set extra string";
static const int   UCS_EVENT_SET_EXTRA_NUM    = 0xFF;
```

C

```C
static const char* UCS_EVENT_SET_EXTRA_STRING = "ucs_event_set extra string";
static const int   UCS_EVENT_SET_EXTRA_NUM    = 0xFF;
```

Reply:
what you have right is OK